heartbeat/redis.in

```diff
@@ -237,6 +237,51 @@ CRM_ATTR_REPL_INFO="${HA_SBIN_DIR}/crm_attribute --type crm_config --name ${INST
 MASTER_HOST=""
 MASTER_ACTIVE_CACHED=""
 MASTER_ACTIVE=""
+CLI_HAVE_AUTH_WARNING=0
+CLI_HAVE_ARG_NO_AUTH_WARNING=0
+CLI_HAVE_ENV_AUTH=0
+
+cmp_redis_version()
```

Comment:
You should be able to use this: https://github.com/ClusterLabs/resource-agents/blob/master/heartbeat/ocf-shellfuncs.in#L597-L604 instead of making your own compare function.

Reply:
Fixed this and some other staff